### PR TITLE
Replaced magic numbers with corresponding enums in print_tensor

### PR DESCRIPTION
**Pull request by sebastiandeutsch**
Just a small one but I've replaced the magic numbers in print_tensor with the corresponding enums from TF_DataType.

### CODE REVIEW

**Comment by nethsix** [comment]
@sebastiandeutsch Reviewed. The code is much cleaner with this!

File: ext/sciruby/tensorflow_c/files/tf_tensor_helper.cc [hunk 1]
```diff
@@ -49,19 +49,19 @@ void print_tensor(TF_Tensor* tensor)
     auto type = TF_TensorType(tensor);
     long long total_elements = 1;
     for (int i = 0; i < dimension_num; ++i) total_elements *= TF_Dim(tensor, i);
-    if (type == 9) {long long* tensor_data = static_cast<long long *>(TF_TensorData(tensor));
+    if (type == TF_COMPLEX) {long long* tensor_data = static_cast<long long *>(TF_TensorData(tensor));
```
Comment (nethsix) [inline ext/sciruby/tensorflow_c/files/tf_tensor_helper.cc]: Should this be TF_INT64?(https://github.com/Arafatk/tensorflow.rb/blob/3d05df3dbbed5c45bac5cc4f9d280c7b79ea6664/ext/sciruby/tensorflow_c/files/tensor_c_api.h)

Comment (Arafatk) [inline ext/sciruby/tensorflow_c/files/tf_tensor_helper.cc]: @nethsix  Yes

**Comment by Arafatk** [comment]
@sebastiandeutsch @nethsix  Thanks

File: ext/sciruby/tensorflow_c/files/tf_tensor_helper.cc (hunk 1 above)
Comment (sebastiandeutsch) [inline ext/sciruby/tensorflow_c/files/tf_tensor_helper.cc]: Yes you're right.

**Pull request merged**